bundle/config/resources/dashboard.go

```diff
@@ -0,0 +1,59 @@
+package resources
```

Comment:
General considerations to review for dashboard properties:
* Top-level run_as should likely apply to dashboards (but should error out for unsupported identities, like it does for DLT)
* Tags should be applied to dashboards
* `is_draft`: it seems okay to allow users to set this. And Iwe shouldn't use this for development mode, or should we?